src/cookies.rs

```diff
@@ -0,0 +1,123 @@
+//! Cookie-related functionality for WebDriver.
+use serde::{Deserialize, Serialize};
+use time::OffsetDateTime;
+use webdriver::command::WebDriverCommand;
+
+use crate::client::Client;
+use crate::error;
+
+/// Type alias for a `cookie::Cookie`
+pub type Cookie<'a> = cookie::Cookie<'a>;
+
+/// JSON representation of a cookie as [defined by WebDriver](https://www.w3.org/TR/webdriver1/#cookies).
+#[derive(Debug, Deserialize, Serialize)]
```

Comment:
It might make sense to expose this type directly instead of converting to/from a `cookie::Cookie`.  The problem with using `cookie::Cookie` is that some fields will not be forwarded to WebDriver, even when set by the user (e.g., `same_site`).